index.js

```diff
@@ -42,8 +42,16 @@ async function assumeRole(params) {
     accessKeyId, secretAccessKey, sessionToken, region, endpoint, customUserAgent: USER_AGENT
   });
 
+  let roleArn = roleToAssume;
+  if (!roleArn.startsWith('arn:aws')) {
+    const identity = await sts.getCallerIdentity().promise();
```

Comment:
now there we're potentially calling more than `sts:assume-role`, would it make sense to include a statement in the `Assuming a role` section of the README about what permissions this action requires?

Reply:
So interesting fact: STS GetCallerIdentity is super fun in that it doesn't require any permissions!  And you can't deny the permission to call it!
https://docs.aws.amazon.com/STS/latest/APIReference/API_GetCallerIdentity.html

Regardless, this API actually gets called already lower down after the credentials are retrieved to export the account ID, so that other actions can use it.

Reply:
😮  TIL!